libkbfs/keybase_service_util.go

```diff
@@ -27,7 +29,7 @@ func EnableAdminFeature(ctx context.Context, runMode libkb.RunMode, config Confi
 // serviceLoggedIn should be called when a new user logs in. It
 // shouldn't be called again until after serviceLoggedOut is called.
 func serviceLoggedIn(ctx context.Context, config Config, session SessionInfo,
-	bws TLFJournalBackgroundWorkStatus) {
+	bws TLFJournalBackgroundWorkStatus) (wg *sync.WaitGroup) {
```

Comment:
It seems only tests rely on the returned `wg` for now. But in case in the future we start using this too, perhaps it makes sense to initialize `wg` at beginning to a `sync.WaitGroup{}` so we never return a `nil` pointer here.